htdocs/langs/en_US/admin.lang

```diff
@@ -513,6 +513,9 @@ Field=Field
 ProductDocumentTemplates=Document templates to generate product document
 FreeLegalTextOnExpenseReports=Free legal text on expense reports
 WatermarkOnDraftExpenseReports=Watermark on draft expense reports
+ForceExpenseReportsLineAmountsIncludingTaxesOnly=Force expense report amounts to be input only including taxes
```

Comment:
What do you mean ? Do you mean adding a line with a VAT of 0 is forbidden ?

Reply:
No, the input for price before taxes is disabled, the user must use the input for price including taxes

Reply:
Sorry, still understand goal of option. Can you provide example.

Reply:
Some companies want to force their users to use only the price incluting taxes in expense reports, I thought in conjuction with @FHenry that it could be useful to push a conf that would do that.